streamz/dataframe/core.py

```diff
@@ -324,7 +342,7 @@ def value_counts(self):
 
 
 class Index(Series):
-    _subtype = pd.Index
```

Comment:
So this is set in `is_frame_like` now (on the instance)? Seems a bit obfuscated. Is this class useful at all?

Reply:
Ideally `Index` should have it's own `init` method. But currently it is being initialized in it's parent class(Series) with an if condition to avoid some code duplication. Not sure if this should be avoided.